src/main/java/nc/students/ayaz/model/Video.java

```diff
@@ -1,18 +1,27 @@
-package nc.students.ayaz;
+package nc.students.ayaz.model;
 
 import cobertura.IgnoreDuringCodeCoverage;
 
 import java.util.ArrayList;
 import java.util.List;
 
-class Video {
+public class Video {
 
     private String name;
+
     private List<Comment> comments;
 
-    Video(String videoName) {
+    public Video() {
+        this.comments = new ArrayList<>();
+    }
+
+    public Video(String videoName) {
         this.name = videoName;
-        comments = new ArrayList<>();
+        this.comments = new ArrayList<>();
+    }
+
+    public String getName() {
```

Comment:
We make it public because we let Jackson to serialize field.
http://www.baeldung.com/jackson-field-serializable-deserializable-or-not